install.sh

```diff
@@ -7,14 +7,21 @@ ORACLE_RPM="$(basename $ORACLE_FILE .zip)"
 
 cd "$(dirname "$(readlink -f "$0")")"
 
-dpkg -s bc libaio1 rpm unzip > /dev/null 2>&1 ||
-  ( sudo apt-get -qq update && sudo apt-get --no-install-recommends -qq install bc libaio1 rpm unzip )
+DEPS="bc libaio1 rpm unzip"
+if [ "$_system_version" = "12.04" ]; then
+   DEPS="$DEPS chkconfig"
+fi
+dpkg -s $DEPS > /dev/null 2>&1 ||
+  ( sudo apt-get -qq update && sudo apt-get --no-install-recommends -qq install $DEPS )
+
+# chkconfig expects /sbin/insserv to exist
+sudo ln -s /usr/lib/insserv/insserv /sbin/insserv
```

Comment:
This sudo will need to be revised in the container patch. Maybe I can container-proof it now..